src/rest_api/music_application.py

```diff
@@ -11,6 +8,11 @@
 
 from tasks import check_done, add_bpm, add_emotions, add_metering, add_similarity_features, save_to_db
 import classification.api_helper as music_emotion_classifier
+# Some linting can show this as not being imported
+from celery import chain
+
+from tasks import check_done, add_bpm, add_emotions, add_metering, \
+    add_similarity_features, save_to_db
 from utilities.config_loader import load_config
 from utilities.get_song_id import get_song_id
```

Comment:
Look at the imports.

I don't this comment is needed:
`# Some linting can show this as not being imported`